_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
 - "Has anyone given you any background on 18F or spoken with you in detail about 18F’s history, purpose, and mission?"
     - If not, fill them in; if so, answer questions (if they've got them).
 
-- ["What drew you to 18F?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6qssbuvkhie2)
+- ["Can you tell me a little bit about your background and how 18F fits into that picture?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.6qssbuvkhie2)
```

Comment:
This is really minor, but 18F probably doesn't fit into their background. How about something like "... tell me about your background and how that led you to 18F"?